float64[7] joint_stiffness
---
bool success
string error

